projects/cdk/project.json

```diff
@@ -30,12 +30,7 @@
             "builder": "@nrwl/workspace:run-commands",
             "options": {
                 "parallel": false,
-                "commands": [
-                    "tsc -p ./tsconfig.schematics.json --tsBuildInfoFile null",
-                    "cp ./projects/cdk/schematics/collection.json ./dist/cdk/schematics/collection.json",
-                    "cp ./projects/cdk/schematics/migration.json ./dist/cdk/schematics/migration.json",
-                    "cp ./projects/cdk/schematics/ng-add/schema.json ./dist/cdk/schematics/ng-add/schema.json"
-                ]
+                "commands": ["tsc -p ./tsconfig.schematics.json --tsBuildInfoFile null"]
```

Comment:
"command": "tsc -p ./tsconfig.schematics.json"

Reply:
Should i remove all `--tsBuildInfoFile null` (inside all `tsc`-commands) or only for this line ?

Reply:
✅